src/popups/InfoFPWPopup/InfoFPWPopup.scss

```diff
@@ -0,0 +1,22 @@
+.wkp-infofpw-popup {
+  div {
+    line-height: 1.42857143;
```

Comment:
are you sure of this ?  I don't want somebody complains thatn the line-height are different between popup.

Reply:
In the old WKP, the line-height was generally 1.4... However, we would have a more compact popup without this line-height. Especially, as we use a popup with a max-height now instead of a sidebar with a variable height compared to the old WKP.

I have removed the line-height for both, the infofpw popup and the parks popup (here: replaced by padding).